blocks/i-lens/i-lens.css

```diff
@@ -0,0 +1,274 @@
+.block.i-lens .title h2 {
+    text-align: center;
+}
+
+.block.i-lens {
+    margin-top: var(--space-15px);
+    box-sizing: border-box;
+    outline: none;
+    padding: 0;
+    border: 0;
+    font-size: 100%;
+    vertical-align: baseline;
+}
+
+.block.i-lens .i-lens-container {
+    background-color: var(--background-color-light-grey);
+    border: 0.06rem solid var(--border-color-dark-grey);
+    border-bottom-left-radius: var(--space-15px);
+    border-bottom-right-radius: var(--space-15px);
+}
+
+.block.i-lens .i-lens-header {
+    background-color: var(--primary);
+    padding: var(--space-10px) var(--space-15px);
+}
+
+.block.i-lens .dropdowns {
+    text-align: left;
+    position: relative;
+    width: 100%;
+    padding-right: var(--space-15px);
+}
+
+
+.block.i-lens .dropdowns .dropdown-select {
+    margin-left: var(--space-10px);
+    background-color: var(--color-white);
+    display: inline-block;
+    margin-bottom:  var(--space-5px);
+    position: relative;
+    width: 100%;
+}
+
+.block.i-lens .dropdowns .icon-down-arrow {
+    right: var(--space-5px);
+    position: absolute;
+
+}
+
+.block.i-lens .dropdowns .icon-down-arrow::before {
+    color: var(--color-orange);
+    content: "\e905";
+    font-size: var(--heading-font-size-sxs);
+    font-weight: var(--font-weight-bolder);
+    font-family: icomoon, sans-serif;
+
+}
+
+.block.i-lens .i-lens-body ul li {
+    font: var(--font-weight-regular) var(--body-font-size-m) / var(--body-font-size-xl) Poppins, "Poppins Fallback",  sans-serif;
+    list-style-position: inside;
+}
+
+.block.i-lens .dropdowns .dropdown-toggle {
+    height: 1.9rem;
+    background-color: var(--background-color-transparent);
+    font-size: var(--heading-font-size-sxs);
+    text-transform: uppercase;
+    padding: 0 1.9rem 0 var(--space-10px);
+    border: 0.06rem solid var(--link-color-gray);
+    border-radius: 0;
+    cursor: pointer;
+    width: 100%;
+}
+
+.block.i-lens .dropdowns .dropdown-toggle .dropdown-text {
```

Comment:
text showing little up in the box, it will fix it 
`vertical-align: middle;`
and `color: #314467d9` , it not in style css so add new.